src/app/service/execution/test-suite-execution.service.config.ts

```diff
@@ -0,0 +1,5 @@
+export class TestSuiteExecutionServiceConfig {
+
+  serviceUrl: string;
```

Comment:
even though this doubles the class name, I'd prefer to name the serviceUrl 'testSuiteExecutionServiceUrl' since this is the only string seen when configured in test-editor-web/app.module.ts.